basic-ruby/parses_dinosaurs.rb

```diff
@@ -1,18 +1,24 @@
 require 'csv'
+require 'json'
 require_relative 'dinosaur'
 require_relative 'african_dinosaur'
 
-class ParsesDinasours
+class ParsesDinosaurs < BaseDinosaur
   def initialize(files)
     @files = files
   end
 
+
   def parse
     files.flat_map do |file|
       parse_single_file(file)
     end
   end
 
+  def to_json
```

Comment:
If you're not going to use this, remove it. If you are going to use, but it is not ready yet, remove it from the pull request and add it later.